### ISSUE TITLE
Feature request: init_image functionality with prdgui

### ISSUE THREAD

**Issue by captain-git-planet** [lowfuel/progrockdiffusion#141]
Maybe it has it and I just can't find it (maybe with the extra inputs box?) but I find it useful to see an image in-progress, but can't do any init_image work because that's only addressable in the settings json.

**Comment by captain-git-planet** [comment]
fiddling around, i can pass an init_image settings file with -s <file> in the additional arguments section but still would be nice to just add an init_image to the gui rather than this way

**Comment by KnoBuddy** [comment]
You can do init images. Just use the CLI commands, and put them into the extra Args.

```-u, --useinit:
                        Forces use of geninit.png as an init_image starting at 20% of defined steps.```

I actually don't think this is the most up to date documentation of the commands from the readme, but I am not at my desktop at the moment to check it out. I know there have been commands that are added that aren't yet documented. 

Let me know if that works. I might have some time later to update the readme with some of the new commands, but adding more features to the GUI isn't a top priority. If I add too many it won't be any better than just editing the settings file itself.

Another issue is that anything more I add to the GUI I want to be hidden under an advanced tab and that is an entirely new programming problem I need to solve.

**Comment by captain-git-planet** [comment]
I tried -u in the past but I wanted to have some more fine control over the steps, I guess I am not sure if the -u command (skips 20% of steps) supercedes the skip_steps in the gui

**Comment by KnoBuddy** [comment]
When I have a chance to look at the code I will see if there is a better CLI option for you or if -u is the only one. Lowfuel might have the answer for you before then.

I may have time to work on the GUI a little bit Sunday, I guess I could throw something together. I really don't want to add more boxes and features without being able to hide some of them so we'll see what happens.

Thanks for your input.

**Comment by captain-git-planet** [comment]
all good! i appreciate you considering it. now that i've figured out the cli extra args workaround with a custom settings file i'm getting most of the way there anyway :)

**Comment by KnoBuddy** [comment]
You can simply modify the fields in the gui_settings.json file that aren't used in the GUI and it will use those settings. So if you want to always use an init file then you can specify the path in the gui_settings.json and PRD GUI won't touch those fields.

**Comment by lowfuel** [comment]
Yes, -u is (was) originally meant to be used in conjunction with --geninit, the idea was you run with one prompt just to get 20% of the way in, then save that and immediately follow it up with another prompt and --useinit to continue from there. It was one of the first things I added.

I do think it would be good to add an init image field to the gui though! And maybe a separate command line option as well.

**Comment by KnoBuddy** [comment]
Added support in /pull/146.
